tests/functional/lti_certification/test_lti_certification_1_1.py

```diff
@@ -0,0 +1,99 @@
+import time
+
+import oauthlib.common
+import oauthlib.oauth1
+import pytest
+from h_matchers import Any
+
+from lms.models import ApplicationInstance
+from tests.functional.base_class import TestBaseClass
+
+
+class TestLTICertification(TestBaseClass):
+    SHARED_SECRET = "TEST_SECRET"
+    OAUTH_CONSUMER_KEY = "Hypothesis1b40eafba184a131307049e01e9c147d"
+    OAUTH_NONCE = "38d6db30e395417659d068164ca95169"
+    OAUTH_CLIENT = oauthlib.oauth1.Client(OAUTH_CONSUMER_KEY, SHARED_SECRET)
+
+    def test_a_good_request_loads_fine(self, app, lti_params):
+        result = self.lti_launch(app, lti_params, status=200)
+
+        self.assert_response_is_html(result)
+
+        # Check a random parameter to see it's passed to the body
+        assert "tool_consumer_instance_guid" in result.text
+        assert lti_params["tool_consumer_instance_guid"] in result.text
```

Comment:
This can be done in a follow-up PR but I think it might be nice to keep `tests/functional/lti_certification/` to strictly test cases that come directly from the certification tool. We can add whatever tests we want outside of this directory. Section 2 of the tests actually does contain a bunch of happy path tests (see https://github.com/hypothesis/lms/issues/932#issuecomment-550422989) so this test could become `test_2_1_launch_as_an_instructor()` and others

Reply:
I was thinking of this to add some of the throw random crap at it type tests